packages/styles/stories/Components/List/list/standard/search-results.example.html

```diff
@@ -0,0 +1,67 @@
+<div style="width: 400px">
+    <ul class="fd-list fd-list--navigation fd-list--search-results" role="list">
```

Comment:
The roles and aria attrs need to be updated a bit to match the a11y requirements